spec/features/keycut_spec.rb

```diff
@@ -39,7 +39,6 @@
 
     it 'adds story (a)', js: true do
       wait_page_load
-      click_on 'Chilly Bin'
```

Comment:
Why was this needed?

Reply:
That was necessary because, when we click on Chilly Bin the keycut stops working. To be honest I don't know why this test was passing before (when we used poltergeist). The correct approach is to press the keycut when we don't have anything selected.